### PR TITLE
[HSEARCH] 7.1.0.Final Release

### PR DESCRIPTION
**Pull request by marko-bekhta**
So that we have something once we are ready to release 🙈

I've taken the `== What's new compared to Hibernate Search 7.0` part from the hibernate.org 7.1 page

### CODE REVIEW

File: posts/MarkoB/2024-02-28-hibernate-search-7-1-0-Final.adoc [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+:hsearch-jira-version-id: 32265
+---
+
+We are pleased to announce the release of Hibernate Search 7.1.0.Final.
+
+Compared to Hibernate Search 7.0,
+this release introduces vector search capabilities,
+allows looking up the capabilities of each field in the metamodel,
+adds a new query string predicate,
+simplifies the entity registration in the Standalone POJO mapper
```
Comment (yrodiere) [inline posts/MarkoB/2024-02-28-hibernate-search-7-1-0-Final.adoc]:
```suggestion
simplifies the entity registration in the Standalone POJO mapper,
```

File: posts/MarkoB/2024-02-28-hibernate-search-7-1-0-Final.adoc [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+for more information.
+
+[[new-since-last-cr]]
+== What's new compared to Hibernate Search 7.0.0.CR1
+
+* link:{hsearch-jira-url-prefix}/HSEARCH-5083[HSEARCH-5083]Upgrade to AWS SDK 2.24.1.
+* link:{hsearch-jira-url-prefix}/HSEARCH-5091[HSEARCH-5091]Upgrade to Elasticsearch *client* 8.12.2.
+* link:{hsearch-jira-url-prefix}/HSEARCH-5087[HSEARCH-5087]Add compatibility with OpenSearch 2.12.0.
+* link:{hsearch-jira-url-prefix}/HSEARCH-5039[HSEARCH-5039]Fix an issue with knn predicates that require multi-tenant filters.
+* link:{hsearch-jira-url-prefix}/HSEARCH-5088[HSEARCH-5088]Limit OpenSearch vector search capabilities to 2.9+ versions.
```
Comment (yrodiere) [inline posts/MarkoB/2024-02-28-hibernate-search-7-1-0-Final.adoc]:
```suggestion
* link:{hsearch-jira-url-prefix}/HSEARCH-5083[HSEARCH-5083]: Upgrade to AWS SDK 2.24.1.
* link:{hsearch-jira-url-prefix}/HSEARCH-5091[HSEARCH-5091]: Upgrade to Elasticsearch *client* 8.12.2.
* link:{hsearch-jira-url-prefix}/HSEARCH-5087[HSEARCH-5087]: Add compatibility with OpenSearch 2.12.0.
* link:{hsearch-jira-url-prefix}/HSEARCH-5039[HSEARCH-5039]: Fix an issue with knn predicates that require multi-tenant filters.
* link:{hsearch-jira-url-prefix}/HSEARCH-5088[HSEARCH-5088]: Limit OpenSearch vector search capabilities to 2.9+ versions.
```

**Review by yrodiere — CHANGES_REQUESTED** [review]
Thanks; I noticed a few typos but otherwise LGTM.

**Comment by marko-bekhta** [comment]
Thanks for taking a look! 😃

**Pull request merged**